docs/sources/get-started/_index.md

```diff
@@ -49,3 +51,5 @@ You can modify your time range in two ways:
 ### What do you think?
 
 Please [share your feedback](https://forms.gle/1sYWCTPvD72T1dPH9) and help make Explore Logs better.
+
+![Screenshot of Explore Logs landing page](../images/explore-logs-features.jpeg)
```

Comment:
I think this screenshot is too busy for documentation.  It's the sort of thing I'd expect to see in a blog post or marketing,  not user documentation.